README.md

```diff
@@ -11,3 +11,38 @@ TODO: Add detailed app description and demo
 ![architecture diagram](images/architecture-diagram.png)
 [Image Link](https://excalidraw.com/#json=1c_DrTFZCSGprCvJNBYHn,B0sXrISCY8YdKgiTqy9xDA)
 _revised 9/4/2024_
+
+The structure of the event database is as follows:
+ ```json
+ {
+  "Event": {
```

Comment:
Nit: if we want to be consistent with naming convention, I used camelCase, so start with lowercase and new words start with uppercase, no spaces in between.

Reply:
nit, not priority, used snake_case